tests/common.go

```diff
@@ -607,6 +607,24 @@ func PrintCommandOutput(cmnd string) {
 	log.Infof(output)
 
 }
+func PrintSvPoolStatus() {
+	PrintSPCommandOutput("pxctl sv pool show")
```

Comment:
we need to run this command on the node where pool expansion happened. take node as input and run on the node

Reply:
Changed the function to take node and execute sv pool show on the node